packages/patternfly-4/react-table/src/components/Table/base/merge-props.ts

```diff
@@ -16,7 +16,7 @@ function mergePropPair(...props) {
   }
 
   // Avoid mutating the first prop collection
-  return mergeWith(mergeWith({}, firstProps), ...restProps, (a, b, key) => {
+  return mergeWith(mergeWith({}, firstProps), ...restProps, (a: any, b: any, key: any) => {
     if (key === 'children') {
       // Children have to be merged in reverse order for Reactabular
```

Comment:
This is also super weird and maybe worth looking at in the future.. or maybe I just don't understand reactabular well enough :)

Reply:
yea... this is merging cell and cell.props into a single prop object... feel free to ehance in the future!

```
        return React.createElement(
          renderers.cell as createElementType,
          {
            key: `${columnIndex}-cell`,
            ...mergeProps(props, cell && cell.props, transformed)
          },
```